admin_app/src/app/content/page.tsx

```diff
@@ -75,13 +77,22 @@ const CardsPage = () => {
   }>({ message: null, color: undefined });
 
   const [openSidebar, setOpenSideBar] = useState(false);
+  const [openChatSidebar, setOpenChatSideBar] = useState(false);
   const handleSidebarToggle = () => {
+    setOpenChatSideBar(false);
     setOpenSideBar(!openSidebar);
   };
-  const handleSidebarClose = () => {
+  const handleChatSidebarToggle = () => {
     setOpenSideBar(false);
+    setOpenChatSideBar(!openChatSidebar);
+  };
+  const handleChatSidebarClose = () => {
+    setOpenChatSideBar(false);
+  };
+  const handleSidebarClose = () => {
+    setOpenChatSideBar(false);
   };
```

Comment:
Closing search sidebar is broken - need to add:


```suggestion
  const handleSidebarClose = () => {
    setOpenChatSideBar(false);
    setOpenSideBar(false);
  };
```